matching-notification-api/src/main/java/org/phenotips/matchingnotification/storage/internal/DefaultMatchStorageManager.java

```diff
@@ -565,4 +574,20 @@ private void saveMatches(Session session, List<PatientMatch> matches)
             session.save(match);
         }
     }
+
+    @Override
+    public Long getNumberOfRemoteMatches()
+    {
+        Session session = this.beginTransaction();
+        long result = -1;
```

Comment:
it is better to use `null` than -1, after all `Long` is an object which supports this special value